proto/archway/rewards/v1beta1/rewards.proto

```diff
@@ -11,12 +11,6 @@ import "cosmos/base/v1beta1/coin.proto";
 message Params {
   option (gogoproto.goproto_stringer) = false;
 
-  // inflation_rewards_ratio defines the percentage of minted inflation tokens that are used for dApp rewards [0.0, 1.0].
-  // If set to 0.0, no inflation rewards are distributed.
-  string inflation_rewards_ratio = 1 [
```

Comment:
I'm not sure if removal is totally correct, maybe removal + adjustment of field number tags.
Or also marking the field as deprecated.

Reply:
Adjusted field tags. 
As there is no need to mark it deprecated if this ships with mainnet.